guides/v2.1/install-gde/system-requirements_repo-tls1-2.md

```diff
@@ -9,6 +9,10 @@ level3_menu_node: level3child
 level3_subgroup: tls
 version: 2.1
 github_link: install-gde/system-requirements_repo-tls1-2.md
+redirect_from:
```

Comment:
Adding redirects for multiple versions won't work for symlinked files (e.g., 2.2 and 2.3). If you want to redirect, you must break those symlinks and create unique files for 2.2 and 2.3. Since this is a redirect from an empty file, I don't think it's really necessary that we do that. I wouldn't want this example to cause confusion for others going forward however, so if you aren't going to create proper redirects, please remove them.